src/Microsoft.Diagnostics.DebugServices.Implementation/CharToLineConverter.cs

```diff
@@ -4,7 +4,7 @@
 using System;
 using System.Text;
 
-namespace Microsoft.Diagnostics.TestHelpers
+namespace Microsoft.Diagnostics.DebugServices.Implementation
```

Comment:
This was OK as test code, but I am not sure it properly handles multibyte sequences. Why is this needed? And why not just take line input as it and copy as-is? What breaks with line feeds?

Reply:
This has always been used in the REPL console service.  It deals with the .Write() of partial lines and turns them into full lines.  It has been part of SOS for a long time; I used in the Midori debugger.  When we have problems with multibyte UTF sequences, I'll fix it.

Reply:
I don't like that it has been duplicated twice (once in REPL and this new one in the impl assembly) but we don't have a common utility assembly yet for things like this.